packages/nimble-components/src/table/styles.ts

```diff
@@ -117,6 +118,25 @@ export const styles = css`
         position: absolute;
     }
 
+    .column-divider:hover {
```

Comment:
Any reason not to combine the selectors to avoid duplicating the styles? I.e.
```css
.column-divider:hover,
.column-divider.active { ... }

.column-divider.visible,
.header-container:hover .column-divider { ... }
```